README.md

```diff
@@ -142,6 +142,33 @@ We currently support the following versions of the [verilator](https://www.verip
 - `v4.108`: [Fedora 34](https://src.fedoraproject.org/rpms/verilator)
 - `v4.202`
 
+## Frequently Asked Questions
+
+### How do I rerun with --full-stacktrace?
+
+Whereas Chisel accepts command-line arguments, chiseltest exposes the underlying annotation interface.
```

Comment:
Could you add one sentence here explaining why one would need to use this option?
You are doing a good job explaining how to enable full stack trace printing, but I am not sure if most users would understand what the benefit is or in which situation they would want to use it.

Reply:
Being perfectly honest, I don't think you ever want it unless you're hitting a bug in Chisel itself. I can add that here but 🤷‍♂️